src/core/MIGRATION.md

```diff
@@ -62,7 +62,7 @@ We'll start with an overview of how plugins work in the new platform, and we'll
 
 Plugins in the new platform are not especially novel or complicated to describe. Our intention wasn't to build some clever system that magically solved problems through abstractions and layers of obscurity, and we wanted to make sure plugins could continue to use most of the same technologies they use today, at least from a technical perspective.
 
-New platform plugins exist in the `src/plugins` and `x-pack/plugins` directories.
+New platform plugins exist in the `src/plugins` and `x-pack/plugins` directories. _See all [conventions for Elastic plugins](./CONVENTIONS.md)_.
```

Comment:
`Elastic` or `Kibana`?

Reply:
This is intended to mean plugins _built at_ Elastic

Reply:
I remember I asked exactly the same question. Probably it's worth changing the wording or adding an explanation?

Reply:
`Official Elastic plugins`? `Elastic provided plugins`?

Reply:
Updated to add "first-party"